ibis/omniscidb/client.py

```diff
@@ -989,16 +929,18 @@ def create_view(self, name, expr, database=None):
         statement = ddl.CreateView(name, select, database=database)
         self._execute(statement)
 
-    def drop_view(self, name, database=None):
+    def drop_view(self, name, database=None, force: bool = False):
```

Comment:
is force a common name across other back ends for this?

Reply:
yes, it is a common name for operations like drop, drop_table, drop_view and drop_database across other backends